cheatsheets/HTML5_Security_Cheat_Sheet.md

```diff
@@ -128,6 +128,23 @@ It is possible to have a [fine-grained control](https://html.spec.whatwg.org/mul
 - In old versions of user agents where this feature is not supported, this attribute will be ignored. Use this feature as an additional layer of protection or check if the browser supports sandboxed frames and only show the untrusted content if supported.
 - Apart from this attribute, to prevent Clickjacking attacks and unsolicited framing it is encouraged to use the header `X-Frame-Options` which supports the `deny` and `same-origin` values. Other solutions like framebusting `if(window!==window.top) { window.top.location=location;}` are not recommended.
 
+# Credential and Personally Identifiable Information (PII) Input hints
+
+- Protect the input values from being cached by the browser. 
+
+Example: access a financial account from a public computer. Even though one is logged-off, the next person who uses the machine can log-in because the browser autocomplete functionality. To mitigate this, we tell the input fields not to assist in any way.
```

Comment:
```suggestion
> Access a financial account from a public computer. Even though one is logged-off, the next person who uses the machine can log-in because the browser autocomplete functionality. To mitigate this, we tell the input fields not to assist in any way.
```

Reply:
One of our penetration testers pointed this out, and we fixed it. 
https://www.makeuseof.com/tag/browser-autofill-privacy-risk-heres-turn-off/
https://twitter.com/anttiviljami/status/816585860661518336

Reply:
I think that it can be useful for the reader to have these links into the section as an additional information to understand the described issue thanks to a visual example.